materialsLibrary/src/custom/customMaterial.ts

```diff
@@ -21,6 +21,7 @@ export class ShaderSpecialParts {
     public Fragment_Begin: string;
     public Fragment_Definitions: string;
     public Fragment_MainBegin: string;
+    public Fragment_MainEnd: string;
```

Comment:
It seems `pbrCustomMaterial` has not been updated in the same way than `CustomMaterial` with `Fragment_MainEnd`?